providers/mlx5/mlx5.c

```diff
@@ -1386,6 +1386,10 @@ static struct verbs_context *mlx5_alloc_context(struct ibv_device *ibdev,
 		}
 	}
 
+	context->cq_uar = mlx5_attach_dedicated_uar(&v_ctx->context,
```

Comment:
Why is it dedicated?

Reply:
This UAR is dedicated for all CQs in the given context comparing other potential customers for such a UAR (QP, DEVX).

Reply:
CQ can share with anything except BF users...

Reply:
Right, this can be some candidate for further optimization to share with QPs with NC mode for example, for now it follows current logic to be for CQs.